app/src/main/java/space/taran/arknavigator/mvp/model/repo/index/ResourceMeta.kt

```diff
@@ -19,7 +19,10 @@ data class ResourceMeta(
     val kind: ResourceKind?,
     val extra: ResourceMetaExtra?) {
 
+
```

Comment:
Unnecessary changes

Reply:
Thank you for that perceived unneeded change; This has been rolled back in the latest commit.